packages/components/src/Table/Table.module.scss

```diff
@@ -20,10 +20,6 @@ $row-height-data-variant: 48px;
   &:focus {
     text-decoration: none;
   }
-
-  &.headerRowCellButtonReversed {
```

Comment:
This never existed so icons would have been dark purple on a purple bg - does make me question if this was ever a problem or even if tables are used on reverse backgrounds? Noting they can also set color on the icon itself so maybe was never a problem?